Makefile

```diff
@@ -7,10 +7,10 @@ LLVM_PROJ_DIR?=$(ROOT_DIR)/src/llvm-project
 # Windows needs munging
 ifeq ($(OS),Windows_NT)
 
-PREFIX?=c:/wasi-sdk
+PREFIX=c:/wasi-sdk
```

Comment:
I'd prefer to leave the `?` in here so these settings can be overridden downstream